server/bigtable.go

```diff
@@ -17,19 +17,31 @@ package server
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	"cloud.google.com/go/bigtable"
 	"github.com/datacommonsorg/mixer/util"
 	"golang.org/x/sync/errgroup"
 )
 
+// TokenType is the type of the token used for each BT row.
+type TokenType int
+
+const (
+	// TokenTypeDcid means DCID is used as BT row token.
+	TokenTypeDcid TokenType = iota
+	// TokenTypeLastItem means the last item in the key string is used as BT row token.
+	TokenTypeLastItem
+)
+
 // bigTableReadRowsParallel reads BigTable rows in parallel,
 // considering the size limit for RowSet is 500KB.
 func bigTableReadRowsParallel(
 	ctx context.Context,
 	btTable *bigtable.Table,
 	rowSet bigtable.RowSet,
 	action func(string, []byte) (interface{}, error),
+	tokenType TokenType,
```

Comment:
i think it does the same thing as the opts param below where i call RemoveKeyPrefix(), so can remove opts?

Reply:
Done.